### PR TITLE
force 45 minute talks and add venue request

### PR DESCRIPTION
**Pull request by gangsta**

### CODE REVIEW

**Comment by kvarak** [comment]
Looks good

File: README.md [hunk 1]
```diff
@@ -43,6 +43,19 @@ Create PR to Repository with Following Changes:
   - With Presentation Link    - [x] *2018-01-15* - [Basic Kubernetes](https://linktopresentation.com) by [Karen Har-yan](talks/Karen-Haryan.md)
   - Without Presentation Link - [x] *2018-01-15* - Basic Kubernetes by [Karen Har-yan](talks/Karen-Haryan.md)
 - Note! Meetups should differ at least 2 weeks. (No Meetups from December 15 - January 15, June 10 - August 31)
+- Good To Know:
+  - Temporary 45 Minute Talk is only option.
```
Comment (kvarak) [inline README.md]: Would rephrase that to: "Temporarily we can only allow 45 minute talks."

**Comment by kvarak** [comment]
Comment on phrasing

File: README.md (hunk 1 above)
Comment (gangsta) [reply README.md]: updated should be good now

**Review by velith — APPROVED** [review]
